ops/ansible/playbooks-ccs/roles/update_jenkins/tasks/install_tools.yml

```diff
@@ -0,0 +1,63 @@
+---
+###
+# Installs additional build and deploy tools
+###
+
+- name: Install tfenv to manage terraform
+  git:
+    repo: https://github.com/tfutils/tfenv.git 
+    dest: /usr/bin/.tfenv
+
+- name: Create symlinks for tfenv
+  file:
+    src: /usr/bin/.tfenv/bin/tfenv
+    dest: /usr/bin/tfenv
+    mode: '0755'
+    state: link
+    
+- name: Link tfenv tf wrapper. 
+  file:
+    src: /usr/bin/.tfenv/bin/terraform
+    dest: /usr/bin/terraform
+    mode: '0755'
+    state: link
+
+- name: Install terraform 
+  shell: tfenv install {{ tf_version }}
+  
+- name: Install packer
+  unarchive:
+    src: https://releases.hashicorp.com/packer/1.6.6/packer_1.6.6_linux_amd64.zip
+    dest: /usr/bin
+    remote_src: yes
+
+- name: Install - AWS CLI
+  pip:
+    name: "awscli"
+
+- name: Create .aws directory in the home directory
+  file:
+    path: "{{ jenkins_home }}/.aws/"
+    state: directory
+    owner: "jenkins"
+    group: "jenkins"
+    mode: 0755
+
+- name: Copy the aws config file to the box
+  become: yes
+  template:
+    src: ../templates/aws_config.j2
+    dest: "{{ jenkins_home }}/.aws/config"
+    owner: "jenkins"
+    group: "jenkins"
+    mode: 0600
+
+- name: Install - jq
+  yum:
+    name: 
+      - jq
+
+- name: Install nfs-utils
+  yum:
+    name:
+      - nfs-utils
```

Comment:
I think these yum tasks can be combined.

Reply:
Yup. Also, we should replace `nfs-utils` with `amazon-efs-utils` as we are mounting EFS file systems and I would like to secure it via IAM roles in the future. But again, I copied and pasted this from the previous existing role.